docs/tutorials/field_type/7_allow_to_add_and_edit_field_in_back_office.md

```diff
@@ -0,0 +1,115 @@
+# Allow to add and edit Field in Back Office
```

Comment:
`Allow adding and editing the Field in Back Office`
(ideally also in the file name)